pennylane/ops/qubit/qchem_ops.py

```diff
@@ -43,6 +49,7 @@ class SingleExcitation(Operation):
 
     * Number of wires: 2
     * Number of parameters: 1
+    * Number of dimensions per parameter: (0,)
```

Comment:
The property is defined [here](https://github.com/PennyLaneAI/pennylane/blob/1558f7d561b0cc709630e2f9d030101afa873b36/pennylane/operation.py#L971).
I agree that it is not explained super-detailed yet. Probably we should introduce the broadcasting-related properties in more detail somewhere.
So far this has not happened because broadcasting mostly is going to be used internally in `param_shift` in 0.25.0 still. But as we support more and more operations, it would be nice to have a proper introduction to "How to make operations and circuits broadcastable".

Reply:
I get it now. A small description might be beneficial but not essentials for now.